ravenframework/InputTemplates/TemplateBaseClass.py

```diff
@@ -26,10 +26,10 @@
 import copy
 # external libraries
 # RAVEN libraries
-frameworkDir = os.path.normpath(os.path.join(os.path.dirname(__file__), '..'))
-sys.path.append(frameworkDir)
-from utils import xmlUtils
-from BaseClasses import MessageUser
+#frameworkDir = os.path.normpath(os.path.join(os.path.dirname(__file__), '..'))
+#sys.path.append(frameworkDir)
```

Comment:
We can probably remove these lines; would they be instructive to leave in?

Reply:
I'll remove them.